api.md

```diff
@@ -82,13 +82,34 @@ partitioning (e.g., for a second time partition) or hash partitioning.
 converted to a hypertable (via `create_hypertable`), but must similarly
 be run only on an empty hypertable.
 
-**Space partitions**: The use of additional partitioning is a very
-specialized use case.  **Most users will not need to use it.**
+**Space partitions**: Using space partitions is highly recommended
+for [distributed hypertables](#create_distributed_hypertable) to achieve
+efficient scale-out performance. For [regular hypertables](#create_hypertable)
+that exist only a single node, the use of additional partitioning is a
+specialized use case and not recommended for most users.
 
 Space partitions use hashing: Every distinct item is hashed to one of
 *N* buckets.  Remember that we are already using (flexible) time
 intervals to manage chunk sizes; the main purpose of space
-partitioning is to enable parallel I/O to the same time interval.
+partitioning is to enable parallelization across multiple 
+data nodes (in the case of distributed hypertables) or
+across multiple disks within the same time interval
+(in the case of single-node deployments).
+
+### Parallelizing queries across multiple data nodes [](add_dimension-multi_node)
```

Comment:
Curious if we should move this sections somewhere else, like performance or optimization

Reply:
I suggest to discuss and consider this outside this PR.